apiaudio/api_resources/voiceCloner.py

```diff
@@ -0,0 +1,136 @@
+from typing import Literal, Optional, List, Union
+
+from apiaudio.api_request import APIRequest
+
+
+class UsersManager(APIRequest):
+    """
+    CRUD interface for Users resource.
+
+    ...
+
+    Notes:
+    - A user should represent a single person/voice
+    - Verticals are assigned to a user
+    - Uploaded files belong to a user
+    - Trained models belong to a user
+
+    """
+
+    OBJECT_NAME = "VC_Users"
+    resource_path = "/ulvc2/users"
+
+    class User(object):
+        """
+        Representation of a voice cloner user.
+        """
+
+        def __init__(
+            self,
+            id: str,
+            name: Optional[str],
+            gender: Optional[Literal["male", "female", "non-binary", ""]],
+            availableVerticals: Optional[list],
+            selectedVerticals: Optional[list],
+            metadata: Optional[dict],
+            voiceCloningEnabled: Optional[bool],
+            createdAt: str,
+            **kwargs,
+        ):
+            self.id = id
+            self.name = name
+            self.gender = gender
+            self.available_verticals = availableVerticals
+            self.selected_verticals = selectedVerticals
+            self.metadata = metadata
+            self.voice_cloning_enabled = voiceCloningEnabled
+            self.created_at = createdAt
+
+        def get_recording_session(self) -> object:  # session object
+            pass
+
+        def add_recording(self, audio: bytes, utterance: object) -> None:
+            pass
+
+        def trigger_model_training(
+            self, name: str = "", verticals_to_use: list = []
+        ) -> None:
+            pass
+
+    @classmethod
+    def create(
```

Comment:
In an ideal world you should create the user object in the init method. I assume having a create function here is just to keep the same method names as other API audio objects?

 You could go as far to make this a 'factory' method - but its not really needed here.

Reply:
My thinking here was: 
the manager is a CRUD interface to the API, so create -> POST method, and the response of that request is parsed into User object. 

Another thing is camelCase in HTTP fields and this_shit in Python 😭